src/instructlab/lab.py

```diff
@@ -1052,9 +1052,10 @@ def train(
         )
 
         training_results_dir = "./training_results"
-        os.makedirs(training_results_dir, exist_ok=True)
 
         final_results_dir = training_results_dir + "/final"
+        if os.path.exists(final_results_dir):
+            shutil.rmtree(final_results_dir)
```

Comment:
I'm a bit uncomfortable blindly removing this whole directory. I'd rather only operate on exactly the files we know are relevant to the current run of `ilab` and not make assumptions about what can be removed otherwise. I just want to avoid any surprising destructive behavior.

Reply:
Actually looking at this again I think I can just remove these two lines. The large model files that it contained is now moved instead of copied so this directory isn't taking up a lot of space.